src/Stack/Dot.hs

```diff
@@ -67,15 +71,23 @@ data DotOpts = DotOpts
     -- ^ Use global hints instead of relying on an actual GHC installation.
     }
 
+data ListDepsFormat = ListDepsText { listDepsSep :: !Text
```

Comment:
I'm personally not a fan of records on sum types, it creates partial accessors that can too easily lead to bugs. How about a helper type that is used by both `ListDepsText` and `ListDepsTree`?

Reply:
Makes sense, I will update the PR and let you know.